src/components/filter/EventsFilterPanel.tsx

```diff
@@ -45,60 +53,79 @@ function EventsFilterPanel() {
 	const eventsStore = useWorkspaceEventStore();
 	const eventDataStore = useEventsDataStore();
 	const filterStore = useEventsFilterStore();
+	const { autocompletes, saveAutocompletes } = useFilterAutocompletesStore();
+	const { addHistoryItem } = useFiltersHistoryStore();
 
 	const [showFilter, setShowFilter] = React.useState(false);
 
-	const [filter, setFilter] = React.useState<EventFilterState | null>(filterStore.filter);
 	const [currentFilterValues, setCurrentFilterValues] = React.useState<CurrentFilterValues | null>(
 		getDefaultCurrentFilterValues(filterStore.filter),
 	);
 
 	React.useEffect(() => {
-		setFilter(filterStore.filter);
 		setCurrentFilterValues(getDefaultCurrentFilterValues(filterStore.filter));
 	}, [filterStore.filter]);
 
 	const onSubmit = React.useCallback(() => {
-		if (filter) {
-			eventsStore.applyFilter(filter);
+		if (filterStore.filter) {
+			// eslint-disable-next-line @typescript-eslint/no-unused-vars
+			const { status, ...restFilters } = filterStore.filter;
```

Comment:
add `ignoreRestSiblings: true` in eslintrc for rule '@typescript-eslint/no-unused-vars' and this will not throw error if desctructered property is omitted